stages/org.osbuild.selinux.meta.json

```diff
@@ -20,8 +20,17 @@
   "schema_2": {
     "options": {
       "additionalProperties": false,
-      "required": [
-        "file_contexts"
+      "oneOf": [
```

Comment:
This should be `anyOf`, otherwise the validation fails when both are specified.

Reply:
fixed